docs/data/advanced-components/overview.md

```diff
@@ -90,9 +90,9 @@ When you purchase a commercial license, you'll receive your license key by email
 This key removes all watermarks and console warnings.
 
 :::warning
-⚠️ All licenses purchased after **May 13, 2022** are _v2_ licenses.
+⚠️ Licenses purchased after **May 13, 2022** are only compatible with MUI X version `5.11` or later.
```

Comment:
We decided that the `:::warning` sections did not need the icon (I can find the discussion on the core if you want)

Reply:
```suggestion
Licenses purchased after **May 13, 2022** are only compatible with MUI X version `5.11` or later.
```